Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -817,6 +817,8 @@ def addStandardSequences(self):
         try:
             if len(self.stepMap):
                 self.loadAndRemember(self.GeometryCFF)
+                if (self.GeometryCFF == 'Configuration/StandardSequences/GeometryRecoDB_cff'):
+                    print("Warning: Default GeometryRecoDB_cff is used. It may not work with your process.")
```

Comment:
@cms-sw/alca-l2 
Do you think this warning might cause confusion? It will appear when a DB condition is used or when no geometry is specified. The goal is to ensure users are aware of the Global Tag (GT) and Geometry they are using.

Reply:
Thank you @srimanob 
As a minimal line of warning that just certifies that there is something  haswhich has to be checked this may work. 
But I think that a user who reads the warning message and does not know about this PR may wonder why the default GeometryRecoDB_cff was used, and what's wrong with it: I would add a line that specifies why this happened and what should be done to fix it.

Reply:
Hi Phat, Andrea,
what is exactly the pitfall that you are trying to warn the user about here?
Isn't taking the reco geometry from DB the recommended way (somehow guaranteed by using the right Global Tag) in the vast majority of the cases (and eventually all of them when the phase-2 geometry is put in DB)?
I think the number of false positive here would be very high.

Reply:
How about:
`Warning: The default GeometryRecoDB_cff is being used. If you are using a DB geometry, you can ignore this warning. However, if no geometry is specified in your cmsDriver command, you may need to verify your configuration.`

Of I can put `if` just to trigger this warning only when no geometry is specify. Maybe that can be done.

Reply:
Fixed. Warning will show up only when no geometry is used in cmsDriver.